unlock-app/public/template.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Unlock</title>
+  <link rel="stylesheet" href="template.css">
+</head>
+
+<body>
+  <div id="root">
+    <div class="container">
+      <div class="item">
+        <!-- Left -->
+      </div>
+
+      <div class="item content">
+        <!-- Content -->
+
+        <header class="top">
+          <h1>Creator Dashboard</h1>
+          <a class="icon"><img src="./images/icons/about.svg" /></a>
+          <a class="icon"><img src="./images/icons/jobs.svg" /></a>
+          <a class="icon"><img src="./images/icons/github.svg" /></a>
+        </header>
+
+        <div class="account">
+          <header class="account__head">
+            <h2>Account</h2>
+            <span class="network_info data">Rinkeby Test Network</span>
+          </header>
+
+          <div class="account__details">
```

Comment:
Beautifully clear HTML. There is an argument for using a table here because it's actually tabular data - but my only real concern is that it could be hard for an accessible browser to read this. I don't know if that's true. It would be nice to check that out sometime soon.